### ISSUE TITLE
make ARCH=armhf failed

### ISSUE THREAD

**Issue by simson** [daniel-thompson/pinebook-pro-debian-installer#4]
It seems that the installation using armhf failed due to the linux-image-pinebook pro is missing within the repository for armhf.

Maybe I missed something ?

Thanks for the great work : it works so great for arm64, but i'd like to use chromium-winedive which is only available on armhf.

Please give me some hint on this

**Comment by daniel-thompson** [comment]
> Please give me some hint on this

I can offer two!

The first is simply that the failure to install is a regression. It was introduced when I updated the kernel installation routines so it can adopt a kernel held in an apt repo (missing: `dpkg --add-architecture arm64` which is needed to install the arm64 kernel inside a armhf userspace). A `git pull` and usual umount and try again should fix this.

The second is that just as we can use `dpkg --add-architecture` to install a 64-bit kernel in a 32-bit userspace we can also use a similar trick to install a 32-bit chromium into a 64-bit userspace! The idea is mention in the thread for the installer (https://forum.pine64.org/showthread.php?tid=8487&highlight=dpkg+--add-architecture ). I tested the concept using firefox but chromium should work too. For chromium there would be no need to install epiphany and remove firefox first. Just copy the last three lines replacing the package names as needed...

Either way it would be great to see a post on the forum detailing your experience with widevine! Good luck.
